mapsforge-core/src/main/java/org/mapsforge/core/util/Parameters.java

```diff
@@ -49,8 +50,12 @@ public enum SymbolScaling {ALL, POI}
 
     /**
      * Use AwtLuminanceShadingComposite or AlphaComposite.
+     * (2024: Deprecated, should not be used)
      */
-    public static boolean LUMINANCE_COMPOSITE = true;
+    public static final boolean LUMINANCE_COMPOSITE = false;
+
+    public static final int HILLSHADING_MAGNITUDE_DEFAULT = 128;
+    public static final int HILLSHADING_COLOR_DEFAULT = 0xff000000;
```

Comment:
@Sublimis These are not parameters, but constants. They should be moved to the class that uses them.